tests/integration/targets/setup_rabbitmq/tasks/ubuntu.yml

```
@@ -37,7 +37,7 @@
 
 - name: Install RabbitMQ Server
   apt:
-    deb: https://ansible-ci-files.s3.us-east-1.amazonaws.com/test/integration/targets/setup_rabbitmq/rabbitmq-server_3.8.14-1_all.deb
+    deb: https://s3.amazonaws.com/ansible-ci-files/test/integration/targets/setup_rabbitmq/rabbitmq-server_3.8.14-1_all.deb
```

```suggestion
    deb: https://s3.amazonaws.com/ansible-ci-files/test/integration/targets/setup_rabbitmq/rabbitmq-server_3.8.14-1_all.deb
```
Could you please remove this change also? The guys are going to fix CI with a dedicated PR

---

Reverted